packages/api-clients/open-api/m2mGatewayApi.yml

```
@@ -4059,6 +4059,94 @@ paths:
               $ref: "#/components/headers/RateLimitRemainingHeader"
             X-Rate-Limit-Interval:
               $ref: "#/components/headers/RateLimitIntervalHeader"
+  /eservices/{eserviceId}/descriptors/{descriptorId}/publish:
+    post:
+      tags:
+        - eservices
+      summary: Publish the selected descriptor.
+      operationId: publishDescriptor
+      description: Publishes the specified descriptor
```

Consistency with the rest of the M2M spec
```suggestion
      summary: Publish an E-Service Descriptor in Draft state
      operationId: publishDescriptor
      description: Publish an E-Service Descriptor that is in Draft state
```